jujugui/static/gui/src/app/components/deployment-bar/_deployment-bar.scss

```diff
@@ -32,15 +32,26 @@
         @extend %floating-panel;
         display: inline-block;
         margin-left: 10px;
+
+        .button--inline-deployment {
+            padding-left: 10px;
+            padding-right: 10px;
+
+            &:first-child {
+                border-top-right-radius: 0;
+                border-bottom-right-radius: 0;
+            }
+
+            &:last-child {
+                border-top-left-radius: 0;
+                border-bottom-left-radius: 0;
+            }
+        }
     }
 
-    &__install-button {
-        @extend %fade-in;
-        @extend %floating-panel;
-        display: inline-block;
-        margin-left: 10px;
-        padding: 10px;
-        color: $cool-grey;
+    .button--inline-neutral {
```

Comment:
No I have replaced the Vanilla buttons with the new buttons in `_generic-button.scss`. They want all button styling to be standardised.